lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -584,6 +663,14 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
     experiment_partitions = get_split_user_partitions(course.user_partitions)
     group_configs_header = [u'Experiment Group ({})'.format(partition.name) for partition in experiment_partitions]
 
+    cert_header = [
+        'Certificate Eligible',
+        'Certificate Delivered',
+        'Enrollment Track',
+        'Verification Status',
```

Comment:
These two do are not related to certificates. You should add them directly to the header. And I would prefer `Enrollment Track` and `Verification Status` to come first.